setup.py

```diff
@@ -28,7 +28,7 @@
     package_dir={"": "src"},
     install_requires=[
         "amazon-braket-schemas>=1.21.3",
-        "amazon-braket-default-simulator>=1.25.0",
+        "amazon-braket-default-simulator @ git+https://github.com/amazon-braket/amazon-braket-default-simulator-python.git@multi2#egg=amazon-braket-default-simulator",  # noqa E501",
```

Comment:
reminder to update after https://github.com/amazon-braket/amazon-braket-default-simulator-python/pull/264 is merged and released